JSDemos/Demos/Toast/Overview/Vue/ProductItem.vue

```diff
@@ -1,38 +1,28 @@
 <template>
   <li>
-    <img :src="product.ImageSrc"><br>
+    <img :src="product.ImageSrc"/><br/>
     <div id="name">{{ product.Name }}</div>
-    <DxCheckBox
-      :on-value-changed="checkAvailability"
-      text="Available"
-    />
+    <DxCheckBox :on-value-changed="onValueChanged" text="Available"/>
```

Comment:
Maybe it is more correct to use event subscription using the following syntax: @value-changed